.github/scripts/generate_manifests.bash

```diff
@@ -16,12 +16,12 @@ module use "$yamf_module_path"
 module load "$yamf_module_name"
 
 # This command finds all the leaf directories: see https://unix.stackexchange.com/questions/68577/find-directories-that-do-not-contain-subdirectories/203991#203991
-dir_without_subdir=$(find "$start_dir" -type d -links 2 ! -empty)
+dir_without_subdir=$(find "$start_dir" -type d -links 2 ! -iwholename "*.git*" ! -empty)
```

Comment:
Man page suggests `-ipath` is more portable than `-iwholename`

Reply:
Ah, I'll update and test that, then :)

Reply:
Done via 1bc38c24106d6bf49ef1410dcb3ccd588349fc78 using GitHub Mobile(?!) on the plane...is this the Mile High Club?